pkg/api/validation/validation.go

```diff
@@ -2012,6 +2021,30 @@ func ValidateSeccompPodAnnotations(annotations map[string]string, fldPath *field
 	return allErrs
 }
 
+const SysctlSegmentFmt string = "[a-z0-9]([_a-z0-9]*[a-z0-9])?"
+const SysctlFmt string = "(" + SysctlSegmentFmt + "\\.)*" + SysctlSegmentFmt
+const SysctlMaxLength int = 253
```

Comment:
Pretty arbitrary value, if I remember right. I like numbers which are a product of two primes.